code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -39,6 +39,8 @@
 
 	voice_of_god_power = 1.4 //Command staff has authority
 
+	dept_jacket = /obj/item/clothing/suit/hooded/wintercoat/security // no hos jacket for some reason
```

Comment:
youre looking for /obj/item/clothing/suit/armor/hos/trenchcoat/winter

Reply:
oh hah, i was mostly looking in wintercoats.dm